test/e2e/import_test.go

```diff
@@ -0,0 +1,85 @@
+//go:build e2e
+// +build e2e
+
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/rancher-sandbox/rancher-turtles/internal/rancher"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
+)
+
+var _ = Describe("Create and delete CAPI cluster functionality should work", func() {
+	var (
+		bootstrapCluster      client.Client
+		rancherClusterHandler *rancher.ClusterHandler
+		capiCluster           *clusterv1.Cluster
+		rancherClusterKey     client.ObjectKey
+	)
+
+	BeforeEach(func() {
+		bootstrapCluster = bootstrapClusterProxy.GetClient()
+		rancherClusterHandler = rancher.NewClusterHandler(ctx, bootstrapCluster)
+		capiCluster = &clusterv1.Cluster{ObjectMeta: metav1.ObjectMeta{
+			Namespace: capiClusterNamespace,
+			Name:      capiClusterName,
+		}}
+		rancherClusterKey = client.ObjectKey{
+			Namespace: capiClusterNamespace,
+			Name:      fmt.Sprintf("%s-capi", capiClusterName),
+		}
+	})
+
+	It("should successfully create a rancher cluster from CAPI cluster", func() {
+		By("Creating a CAPI docker provider cluster setup")
+		Eventually(func() error {
+			return bootstrapClusterProxy.Apply(ctx, fleetCapiTestdata)
+		}, e2eConfig.GetIntervals(bootstrapClusterProxy.GetName(), "wait-controllers")...).Should(Succeed())
+
+		By("Waiting for rancher cluster record to appear")
```

Comment:
should also check that it was imported succesfully

Reply:
Now this works we should:
- get the kubeconfig for the child cluster and check that we can connect.
- Check the status of the Rancher Cluster